shared/api/instance.go

```diff
@@ -418,6 +418,19 @@ type InstanceSource struct {
 	//
 	// API extension: instance_allow_inconsistent_copy
 	AllowInconsistent bool `json:"allow_inconsistent" yaml:"allow_inconsistent"`
+
+	// Source disk size in bytes used to increase the instance's volume size to accommodate the transferred
+	// root disk. This value is ignored if the root disk device is explicitly configured (for conversion).
```

Comment:
> This value is ignored if the root disk device is explicitly configured (for conversion).

Should this say "...ignored if the root disk device has a size explicitly configured (for conversion)"?